client/src/app/slides/motions/motion/motions-motion-slide.component.ts

```diff
@@ -1,14 +1,258 @@
 import { Component } from '@angular/core';
 import { BaseSlideComponent } from 'app/slides/base-slide-component';
 import { MotionsMotionSlideData } from './motions-motion-slide-data';
+import { ChangeRecoMode, LineNumberingMode } from '../../../site/motions/models/view-motion';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { ConfigService } from '../../../core/ui-services/config.service';
+import { DiffService, LineRange } from '../../../core/ui-services/diff.service';
+import { LinenumberingService } from '../../../core/ui-services/linenumbering.service';
+import { ViewUnifiedChange } from '../../../shared/models/motions/view-unified-change';
+import { MotionsMotionSlideObjChangeReco } from './motions-motion-slide-obj-change-reco';
 
 @Component({
     selector: 'os-motions-motion-slide',
     templateUrl: './motions-motion-slide.component.html',
     styleUrls: ['./motions-motion-slide.component.scss']
 })
 export class MotionsMotionSlideComponent extends BaseSlideComponent<MotionsMotionSlideData> {
-    public constructor() {
+    /**
+     * Indicates the LineNumberingMode Mode.
+     * @TODO Read value from the backend
+     */
+    public lnMode: LineNumberingMode = LineNumberingMode.Outside;
+
+    /**
+     * Indicates the Change reco Mode.
+     * @TODO Read value from the backend
+     */
+    public crMode: ChangeRecoMode = ChangeRecoMode.Diff;
+
+    /**
+     * Indicates the maximum line length as defined in the configuration.
+     * @TODO Read value from the backend
+     */
+    public lineLength: number = 90;
+
+    /**
+     * Indicates the currently highlighted line, if any.
+     * @TODO Read value from the backend
+     */
+    public highlightedLine: number;
+
+    /**
+     * Value of the config variable `motions_preamble`
+     * @TODO Read value from the backend
+     */
+    public preamble: string = 'Die Versammlung möge beschließen';
+
+    /**
+     * All change recommendations AND amendments, sorted by line number.
+     */
+    public allChangingObjects: ViewUnifiedChange[];
+
+    public constructor(
+        private configService: ConfigService,
+        private sanitizer: DomSanitizer,
+        private lineNumbering: LinenumberingService,
+        private diff: DiffService
+    ) {
         super();
     }
+
+    /**
+     * Merges amendments and change recommendations and sorts them by the line numbers.
+     * Called each time one of these arrays changes.
+     */
+    private recalcUnifiedChanges(): void {
+        this.allChangingObjects = [];
+
+        if (this.data.data.change_recommendations) {
+            this.data.data.change_recommendations.forEach(change => {
+                this.allChangingObjects.push(new MotionsMotionSlideObjChangeReco(change));
+            });
+        }
+        /*
+        @TODO
+        if (this.amendments) {
```

Comment:
What for attributes of an amendment do you need? Do you need CRs to an amendment?
For the server (somehow like this):
```
def get_amendments_for_motion(motion, all_data):
    amendments = []
    for amendment in all_data.items():  # all_data is a dict: id<->motion object
        if amendment['parent_id'] == motion['id']:
            amendments.append(amendment)
    amendment_data = []
    for amendment in amendments:
        amendment_data.append({
            'needed_arrt1:' amendment['...'],
            ...
        })
    return amendment_data

... in the get_motion_slide ...

return_value['amendments'] = get_amendments_for_motion(motion, all_data["motions/motion"])
```